_topics/en/cluster-mempool.md

```diff
@@ -52,20 +55,102 @@ see_also:
 excerpt: >
   **Cluster mempool** is a proposal to associate each unconfirmed
   transaction in a mempool with related transactions, creating a
-  cluster.  Each cluster of transactions, whether it be a single
-  transaction or several transactions, can be ordered from most
-  desirable to mine to least desirable, allowing operations for adding
-  or removing new clusters to complete fast enough to use them in P2P
-  network code.
+  _cluster_.  Each cluster contains feerate-sorted _chunks_ consisting
+  of one or more transactions.  If clusters are limited in size,
```

Comment:
```suggestion
  of one or more transactions.  If we limit a cluster's size,
```
(otherwise the size sounds exgengous, which I don't think is the intention?)